src/plot_api/plot_api.js

```diff
@@ -1738,8 +1738,8 @@ function addAxRangeSequence(seq, rangesAltered) {
     // subroutine of its own so that finalDraw always gets
     // executed after drawData
     var doTicks = rangesAltered ?
-        function(gd) { return subroutines.doTicksRelayout(gd, rangesAltered); } :
-        subroutines.doTicksRelayout;
+        function(gd) { return Axes.doTicks(gd, Object.keys(rangesAltered), true); } :
+        function(gd) { return Axes.doTicks(gd, 'redraw'); };
```

Comment:
> just calling 'Axes.doTicks' instead of full `doTicksRelayout` which includes drawing the title

Is that sufficient though? If you move the ticks and as a result the title gets pushed away a different amount to avoid them, is that handled correctly?

Reply:
I think this is sufficient. The `drawMainTitle` subroutine 

https://github.com/plotly/plotly.js/blob/e646042b40a4fc123cdd84d4e525ebe64df0c088/src/plot_api/subroutines.js#L428-L441

doesn't attempt to scoot to title away from tick labels.

Reply:
Ah you're right - I missed that this was only about the main title, not the axis titles 👍